build/shade/_dotnet-test.shade

```diff
@@ -37,7 +38,18 @@ default configuration = 'Debug'
         }
 
         testArgs += noParallelTestProjects.Contains(projectName) ? " -parallel none" : "";
-        Dotnet("test" + testArgs, projectFolder);
+
+        try
+        {
+            Dotnet("test" + testArgs, projectFolder);
+        }
+        catch
```

Comment:
There's a risk of it, but there's not much we could do short of matching the exception message here: https://github.com/sakeproject/sake/blob/master/src/Sake.Library/Shared/_exec.shade#L60